backend/src/main/java/pl/pollub/backend/expenses/ExpenseService.java

```diff
@@ -29,12 +30,12 @@ public Expense updateExpense(Long id, Expense updatedExpense) {
                 })
                 .orElseThrow(() -> new RuntimeException("Nie znaleziono wydatku o podanym identyfikatorze: " + id));
```

Comment:
change each RuntimeException to HttpException with suitable http status code (404 in this case)